### ISSUE TITLE
windows path bug

### ISSUE THREAD

**Issue by marath007** [Bluebotlabz/Universal-Hearing-Aid-Software-Downloader#3]
**Describe the bug**

**To Reproduce**
1. run the script
2. Insert path ex: C:\Users\tonys\Downloads\other

**Expected behavior**
file going in the folder

**Script output**
no files

**Script used**
Please specify which script you used

**Desktop (please complete the following information):**
windows 11

**Additional context**

I fixed it by editing the normalize path in libhearingdownloader.py
it seems to be meant for linux. :D

```python
def normalizePath(path, correctWindowsChars=False):
    if (path[-1] != "\\"):
        path += "\\"

    if (correctWindowsChars):
        return (path.replace("|", "-").replace("<", "-").replace(">", "-").replace(":", "-").replace('"', "-").replace("?", "-").replace("*", "-"))
    else:
        return path
```

**Comment by Bluebotlabz** [comment]
Strange, the linux paths should work fine for this...

I'll implement a fix anyway

**Comment by marath007** [comment]
here is the original faulty output for the path
Enter an output directory: C:\Users\tonys\Downloads\other
Confirm download path (C-\Users\tonys\Downloads\other/) [Y/n]

**Comment by Bluebotlabz** [comment]
> here is the original faulty output for the path Enter an output directory: C:\Users\tonys\Downloads\other Confirm download path (C-\Users\tonys\Downloads\other/) [Y/n]

ah, I see the problem...

let me know if this code works:

~~~python3
def normalizePath(path, correctWindowsChars=False):
    if (path[-1] != "/"):
        path += "/"

    if (correctWindowsChars):
        return (path.replace("|", "-").replace("<", "-").replace(">", "-").replace('"', "-").replace("?", "-").replace("*", "-"))
    else:
        return path
~~~

(isn't a fix, should work, but just a test)

**Comment by marath007** [comment]
C:\Users\tonys\Downloads\Universal-Hearing-Aid-Software-Downloader--v1.6.2\-C-\Users\tonys\Downloads\other\--

I mean it works.. lol just a terrifying path

**Comment by marath007** [comment]
"C:\Users\tonys\Downloads\others\" fails
C:\Users\tonys\Downloads\others succeed properly with the new code

**Comment by Bluebotlabz** [comment]
Yeah, I think I see the problem, I'll fix it in the next update which should come out soon

**Comment by Bluebotlabz** [comment]
> C:\Users\tonys\Downloads\Universal-Hearing-Aid-Software-Downloader--v1.6.2-C-\Users\tonys\Downloads\other--
> 
> I mean it works with the old code, lol just a terrifying path

yeah lol

**Comment by Bluebotlabz** [comment]
This has been fixed in [v1.6.3](https://github.com/Bluebotlabz/Universal-Hearing-Aid-Software-Downloader/releases/tag/v1.6.3)
